source/docs/03-metadata.xml

```diff
@@ -1393,156 +1397,470 @@
+                        </figure>
                      </p>
-                     <p>The values of those attributes, however, specify the height and width of the digital resource, the scan of the source, and they are typically given in pixels (see <ptr target="#facsimileElements"/>). In contrast, <gi scheme="MEI">folium</gi> and <gi scheme="MEI">bifolium</gi> are supposed to provide the dimensions of the original source in cm or inches. This makes it possible to combine separate parts of a manuscripts stored in different libraries, which are scanned at different resolutions. In case of <gi scheme="MEI">bifolium</gi> elements, these dimensions apply to the <hi rend="italic">folded</hi> sheet.</p>
-                     <p>Some printed scholarly editions like the <hi rend="italic">Neue Bach-Ausgabe</hi> provide very detailed information about the sizes and binding of individual leaves of a manuscript; with <gi scheme="MEI">foliaDesc</gi> and its children it is possible to encode that information in processable ways, even without providing digitizations of the sources via <gi scheme="MEI">surface</gi>.</p>
+                     <p>A higher level of distinction is also recommended for address stamps:</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample084.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p><specList>
+                        <specDesc key="scoreFormat"/>
+                     </specList></p>
+                     <p>The <gi scheme="MEI">scoreFormat</gi> element is a form of classification. This element is part of <gi scheme="MEI">physDesc</gi> because within the MARC21 standard, the format of the music (score, piano score, etc.) is defined as a physical property.</p>
+                     <p>For describing the <gi scheme="MEI">scoreFormat</gi> a standard value list can be used, e.g., MARC21 (see “20 - Format of music (006/03)”, <ref target="https://www.loc.gov/marc/bibliographic/bd008m.html">https://www.loc.gov/marc/bibliographic/bd008m.html</ref>) or RDA (see <ref target="https://www.rdaregistry.info/termList/formatNoteMus/">https://www.rdaregistry.info/termList/formatNoteMus/</ref>). The values can be kept using the attributes <att>authority</att> and <att>authURI</att>.</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample094.txt" parse="text"/></egXML>
```

Comment:
The referenced `header-sample094.txt`is missing consequently the xi:include fails as no fallback is provided and then the build fails

Reply:
I notice that `header-sample094.txt` is missing. I'll figure out what that should be and what went wrong. Maybe one merge failed and `header-sample080.txt` has been overwritten.

Reply:
Example restored in 902eb8650dea63143d84327dcf69094e50597254